examples/platform/asset_groups.py

```diff
@@ -0,0 +1,191 @@
+    changes = Policy.preview_policy_rank_changes(api, [(policy_top_rank.id, new_policy_position)])
+    if len(changes) == 0:
+        print("No changes to effective policy would occur.")
+    else:
+        print("There are {} changes that would result from moving Policy {} from position {} to position {}."
+              .format(len(changes), policy_top_rank.name, policy_top_rank.position, new_policy_position))
+
+    for i, c in enumerate(changes, 1):
+        print("printing change number {}".format(i))
+        current_top_rank_policy = api.select(Policy, c.current_policy_id)
+        future_top_rank_policy = api.select(Policy, c.new_policy_id)
+        print("{} assets will be affected.".format(c.asset_count))
+        print("The assets affected are:")
+        assets_affected = c.asset_query.all()
+        for a in assets_affected:
+            print("Asset Name: {} - Asset Id {}".format(a.name, a.id))
+        print("\n The currently effective policy for those assets is: name: {}, id: {}".
+              format(current_top_rank_policy.name, current_top_rank_policy.id))
+        print("\n The effective policy after the move would be: name: {}, id: {}".
+              format(future_top_rank_policy.name, future_top_rank_policy.id))
+    print("\n\n Finished preview_policy_rank_change \n\n")
+
+
+def preview_asset_group_changes():
+    """Show how to use the preview asset group function to understand the impact of changes such as changing a query"""
+    print("\n\n preview_asset_group_changes(): Coming Soon \n\n")
+
+
+def main():
+    """This script demonstrates how to use Asset Groups in the SDK and common operations to link to related objects.
+
+    This example does not use command line parsing in order to reduce complexity and focus on the SDK functions.
+    Review the Authentication section of the Read the Docs for information about Authentication in the SDK
+    https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/authentication/
+
+    This is written for clarity of explanation, not perfect coding practices.
+    """
+    # CBCloudAPI is the connection to the cloud.  It holds the credentials for connectivity.
+    # To execute this script, the profile must have an API key with the following permissions.
+    # If you are restricted in the actions you're allowed to perform, expect a 403 response for missing permissions.
+    # Permissions are set on Settings -> API Access -> Access Level and then assigned to an API Key
+    # GM - Group Management - group-management - CREATE, READ, UPDATE, DELETE: For managing asset groups
+    # Device - General Information - device - READ: For getting device information
+    # Policies - Policies - org.policies - READ: For viewing policy information and pre-viewing the impact of changes
+    #    to policy ranking and asset groups.
+
+    # api = CBCloudAPI(profile="YOUR_PROFILE_HERE")
+    api = CBCloudAPI(profile="ASSET_GROUP_DEV_01")
```

Comment:
Should this be swapped out with the commented one as probably not best to reference dev01